src/test/java/org/broadinstitute/hellbender/tools/coveragemodel/CoverageModelCopyRatioEmissionProbabilityCalculatorUnitTest.java

```diff
@@ -0,0 +1,150 @@
+package org.broadinstitute.hellbender.tools.coveragemodel;
+
+import org.apache.commons.math3.util.FastMath;
+import org.broadinstitute.hellbender.tools.exome.sexgenotyper.SexGenotypeData;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+/**
+ * Unit tests for {@link CoverageModelCopyRatioEmissionProbabilityCalculator}
+ *
+ * @author Mehrtash Babadi &lt;mehrtash@broadinstitute.org&gt;
+ */
+public class CoverageModelCopyRatioEmissionProbabilityCalculatorUnitTest extends BaseTest {
```

Comment:
IMO unit tests for any method that corresponds to a specific mathematical formula should always, if possible, include a few asymptotic limits.  Here, the large depth limit should be tractable because the Poisson becomes a Gaussian and therefore the whole thing is Gaussian a of a linear transformation of a Gaussian, which is Gaussian.